Astronomy/Astronomy/Model/JLRCache.m

```diff
@@ -0,0 +1,22 @@
+//
+//  JLRCache.m
+//  Astronomy
+//
+//  Created by Jesse Ruiz on 12/16/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+#import "JLRCache.h"
+
+@implementation JLRCache
+
+- (void)setObject:(id)obj forKey:(id)key{}
```

Comment:
Always put the bracket on the next line to keep it consistent across all your functions.

```swift
-(id)objectForKey:(id)key {
     return [self objectForKey:key];
 }
```